lighthouse-core/config/config-plugin.js

```diff
@@ -141,6 +145,56 @@ class ConfigPlugin {
     };
   }
 
+
+  /**
+   * Extract and validate groups JSON added by the plugin.
+   * @param {unknown} groupsJson
+   * @param {string} pluginName
+   * @return {Record<string, LH.Config.GroupJson>|undefined}
+   */
+  static _parseGroups(groupsJson, pluginName) {
```

Comment:
I wonder if we should be enforcing any extra constraints here, like if there are groups, all audits belong to a group or something

Reply:
My one concern with this would be the case of including manual audits. This doesn't apply to our use case at the moment, but I imagine it may apply to others.

Reply:
That's true, I'm fine with holding as-is to start 👍